tempodb/backend/cache/memcached/memcached.go

```diff
@@ -0,0 +1,60 @@
+package memcached
```

Comment:
We can add a `memcached_test.go` with just a single line:

```
var _ (cache.Cache) = (*Cache)(nil)
```

similarly for redis.